### PR TITLE
fix to get_guess added while true loop and try exception with error an message.

### PR DESCRIPTION
**Pull request by Hockeydan25**
@chrismon88 Hi Chris can you review my fix to the program #1 .

### CODE REVIEW

File: guess_the_number.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from distutils.log import error
```
Comment (chrismon88) [inline guess_the_number.py]: Not sure if we need this line.

File: guess_the_number.py [hunk 2]
```diff
@@ -12,13 +13,18 @@ def configure_range():
...
 def generate_secret(low, high):
     """ Generate a secret number for the user to guess """
+
     return random.randint(low, high)
 
 
 def get_guess():
     """ get user's guess, as an integer number """
-    return int(input('Guess the secret number? '))
+    while True:
```
Comment (chrismon88) [inline guess_the_number.py:20]: Code looks good, double check the spaces

Comment (Hockeydan25) [reply guess_the_number.py:20]: I made corrections to the invalid import and spacing.

**Comment by chrismon88** [comment]
tested with negative and strings. Code passed.

**Comment by Hockeydan25** [comment]
We are leaving branch in tack for today.

**Comment by claraj** [comment]
Looks great team! Excellent collaboration on the review, and code.

**Pull request merged**